pilopt/src/lib.rs

```diff
@@ -215,6 +217,39 @@ fn constant_value(function: &FunctionValueDefinition) -> Option<BigUint> {
     }
 }
 
+/// Deduplicate fixed columns which share the same value.
+/// This uses the `Display` implementation of the function value, so `|i| i` is different from `|j| j`
```

Comment:
Is it that much more expensive to check for syntactic equality? I think Display is very expensive.

Reply:
I can have a go at it, seemed annoying because of sourcemaps.